app/src/main/java/dto/Pokemon.kt

```diff
@@ -1,5 +1,5 @@
 package dto
 
-data class Pokemon(var pokemonId: Int = 0, var name: String="", var description: String="", var types: ArrayList<String>) {
+data class Pokemon(var pokemonId: Int = 0, var name: String = "", var description: String = "", var types: ArrayList<String>) {
     override fun toString() = "$name $description"
```

Comment:
Put spacing before and after the equal sign as seen in this source: https://kotlinlang.org/docs/reference/coding-conventions.html